src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -60,7 +74,11 @@ public Double averageSumOdd(List<Integer> numbers) {
      * Задача: Выбрать мужчин-военнообязанных (от `fromAge` до `toAge` лет)</p>
      **/
     public List<People> manSelectByAge(List<People> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(person -> person.getSex().equals(People.Sex.MAN)
```

Comment:
better use == instead of equals() with enum
https://stackoverflow.com/a/1750453